### PR TITLE
Update to MG benchmark suite

### PR DESCRIPTION
**Pull request by nelsonag**
This PR updates the C5G7 and MG Criticality Benchmark suite to the latest rev of OpenMC.

### CODE REVIEW

File: mg-criticality/make_model.py [hunk 1]
```diff
@@ -303,7 +303,7 @@ def make_model(self):
     def execute(self, quiet=True):
         returncode = openmc.run(output=(not quiet))
         spfile = 'statepoint.' + str(self.batches) + '.h5'
-        sp = openmc.StatePoint(spfile)
-        self.keff = sp.k_combined[:]
+        sp = openmc.StatePoint(spfile, autolink=False)
+        self.keff = sp.k_combined
 
         return returncode
```
Comment (paulromano) [inline mg-criticality/make_model.py]: `openmc.run` doesn't actually return a returncode anymore but will raise `subprocess.CalledProcessError`, so this should be changed to catch the exception and return a non-zero value (or let the exception propagate and catch it elsewhere).

File: mg-criticality/make_model.py [hunk 2]
```diff
@@ -55,7 +53,7 @@ def make_materials(self):
             mats.append(openmc.Material(name=self.mat_names[i]))
             mats[-1].set_density('macro', 1.0)
             mats[-1].add_macroscopic(macros[-1])
-            materials_file.add_material(mats[-1])
+            materials_file += [mats[-1]]
```
Comment (paulromano) [inline mg-criticality/make_model.py]:
```suggestion
            materials_file.append(mats[-1])
```

**Comment by nelsonag** [comment]
Thanks for the review @paulromano, just implemented the changes!

**Comment by paulromano** [comment]
Thanks again for the fixes @nelsonag!

**Pull request merged**